<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recuperar_contra</class>
 <widget class="QDialog" name="recuperar_contra">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingresa tu curp</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Recuperacion de contraseña</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_curp">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
border-bottom: 1px solid #C4C4C4;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_continuar">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>290</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background: #3E64FF;
color: white;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Continuar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Selecciona el tipo de usuario</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="box_usuario">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background: white;
border-bottom: #C4C4C4;</string>
     </property>
     <item>
      <property name="text">
       <string>Medico</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Paciente</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Staff</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <zorder>label</zorder>
    <zorder>label_3</zorder>
    <zorder>label_2</zorder>
    <zorder>line_curp</zorder>
    <zorder>btn_continuar</zorder>
    <zorder>label_11</zorder>
    <zorder>box_usuario</zorder>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Recuperacion de contraseña</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingresa la palabra clave que ingresaste al inicio</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_palabra">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
border-bottom: 1px solid #C4C4C4;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_enviar">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>290</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background: #3E64FF;
color: white;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Enviar</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Recuperacion de contraseña</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_hecho">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>290</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background: #3E64FF;
color: white;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Hecho</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Su nombre de usuario es</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_nombre_usuario">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
border-bottom: 1px solid #C4C4C4;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Su cotraseña es</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_contrasenia">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
border-bottom: 1px solid #C4C4C4;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_8</zorder>
    <zorder>btn_hecho</zorder>
    <zorder>label_9</zorder>
    <zorder>line_nombre_usuario</zorder>
    <zorder>label_7</zorder>
    <zorder>label_10</zorder>
    <zorder>line_contrasenia</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
